### DISCUSSION TITLE
[Q] Apply specific filtering when exporting excel file

### DISCUSSION THREAD

**Discussion by giaotrinh147** [exceljs/exceljs#2424] [Q&A]
## 💬 Questions and Help

Hello , I know we can activate the auto filter in excel file but is there any way to apply the specific filter in excel when the auto filter is on with exceljs  ? Thanks

**Comment by ttaha8** [comment]
Please let me know if you have found a solution for this

**Comment by leonardolombardi07** [comment]
I also want to know this
